tests/pipeline/base.py

```diff
@@ -161,15 +160,6 @@ def get_equity_info(cls):
             end_date=pd.Timestamp('2015-01-01', tz='UTC'),
         )
 
-    def zip_with_floats(self, dates, flts):
```

Comment:
was this just dead code? do we need to replace this with the stuff you added in utils.py?